packages/fxa-content-server/app/scripts/views/survey.jsx

```diff
@@ -34,7 +38,7 @@ const SurveyView = (props) => {
     return () => window.removeEventListener('message', handleIframeTask);
   }, [setShowSurvey, setSurveyComplete, setTimeout]);
 
-  return <>{showSurvey && <Survey {...{ surveyComplete, surveyURL }} />}</>;
+  return <>{showSurvey && <Survey {...{ surveyComplete, surveyURL, onSurveyClose }} />}</>;
```

Comment:
I see this isn't part of your changes, but... is the fragment necessary?

Reply:
I see this a few times below as well, I think you can get away with just `return (condition && <Component />)`, no?

Reply:
Looks like that's the equivalent of `return false` when `showSurvey` is `false` - no complains in the JSX file, but Storybook has to have a React element returned. I can make that change here, will wait on other comments. 👍

Reply:
Ah okay as long as it's intentional! Carry on.